README-pt_br.md

```diff
@@ -0,0 +1,110 @@
+* [ES6](#learning-es6)
+* [Rotas](#learning-routing)
+* [Flux](#learning-flux)
+
+**Você não precisa aprender todos para ser produtivo com o React.** Somente pule para o próximo passo caso você esteja com algum problema que necessite ser resolvido.
+
+Além disso, existem alguns tópicos mencionados frequentemente na comunidade do React que são "pioneiros". Os tópicos a seguir são interessantes, mas são difíceis de entender, são muito menos populares do que os tópicos acima e **não são necessários para a maioria dos aplicativos**.
+* [Estilos Inline](#learning-inline-styles)
+* [Renderizando no servidor](#learning-server-rendering)
+* [Immutable.js](#learning-immutablejs)
+* [Relay, Falcor, etc](#learning-relay-falcor-etc)
+
+## Aprendendo o React em si
+
+É um equívoco comum que diz que você precisa perder bastante tempo configurando as ferramentas para começar a aprender React. Na documentação oficial você encontrará [um template de copiar/colar](https://facebook.github.io/react/docs/getting-started.html#quick-start-without-npm) que você pode salvar em um arquivo `.html` e começar imediatamente. **Nesse momento nenhuma configuração é necessária, e você não precisa começar a estudar configurações adicionais até que você esteja confortável com o básico do React.**
+
+Eu ainda acho que o jeito mais fácil de aprender React é através do [tutorial oficial](https://facebook.github.io/react/docs/tutorial.html).
+
+## Aprendendo `npm`
+
+`npm` é o gerenciador de pacotes do Node.JS e é a maneira mais popular para engenheiros front-end e designers compartilharem seus códigos JavaScript. Ele inclui um módulo chamado `CommonJS` e permite que você instale ferramentas escritas em JavaScript através da linha de comando. Leia [esse post](http://0fps.net/2013/01/22/commonjs-why-and-how/) para uma idéia de como o `CommonJS` é necessário para os navegadores, ou o [CommonJS Spec Wiki](http://wiki.commonjs.org/wiki/Introduction) para mais sobre o API do `CommonJS`.
+
+A maioria dos componentes reutilizáveis, bibliotecas e ferramentas no ecossistema do React estão disponíveis como `módulos CommonJS` e são instalados com o `npm`.
+
+## Aprendendo os JavaScript bundlers
+
+Por algum bom número de razões técnicas os módulos `CommonJS` (ex. tudo no `npm`) não podem ser usados nativamente no navegador. Você precisa de um “bundler” JavaScript para “converter” esses módulos em arquivos `.js` que possam ser incluídos na sua página com uma tag `<script>`.
+
+Exemplos de bundlers JavaScript incluem `webpack` e `browserify`. Ambos são boas escolhas, mas eu prefiro o `webpack` uma vez que ele possui diversas ferramentas que fazem o desenvolvimento de aplicativos maiores mais fáceis. Visto que sua documentação pode ser confusa, eu criei [um template plug-and-play para começar](https://github.com/petehunt/react-webpack-template) e escrevi um [guia 'how-to' para o webpack](https://github.com/petehunt/webpack-howto) para casos de uso mais complexos.
```

Comment:
Empacotadores ao invés de bundlers, se decidir usar no parágrafo anterior, uma vez que já fez a apresentação e explicação do termo.

"(...) mas eu prefiro o `webpack`, uma vez que ele possui várias funcionalidades que facilitam o desenvolvimento de aplicativos maior porte" - features != ferramentas

"(...) eu criei um modelo base e escrevi um guia (...)" - how-to guide pode ser traduzido apenas como guia.